### ISSUE TITLE
Suggestion: Give default values to the public properties of the SekibanAzureOption class

### ISSUE THREAD

**Issue by tkawae** [J-Tech-Japan/Sekiban#196]
Some of the properties of `Sekiban.Infrastructure.Cosmos.SekibanAzureOption` class have default values defined. However, some properties without default values also have default values passed at runtime. To add uniformity to the coding, how about defining the default values in the property declaration whenever possible if the property has a default value?

The following properties are likely to be able to define default values.
- BlobConnectionStringName
- CosmosConnectionStringName
- CosmosDatabase

**Comment by tomohisa** [comment]
I agree. Currently only default was put in FromConfiguration but it should be static const and default value too.
